packages/terminal/src/node/shell-process.ts

```diff
@@ -103,17 +103,22 @@ export class ShellProcess extends TerminalProcess {
     }
 }
 
+export interface MergeProcessEnvOptions {
```

Comment:
Why are we defining two interfaces with the same signature?  i.e. `NormalizeEnvOptions`, and `MergeProcessEnvOptions`,

Reply:
It sounds better API-wise: The functions happen to share a common option field now, but it doesn't mean they actually depend on the same options. Hence why I kept it separated here.

Reply:
Thanks for the explanation !!